supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -79,7 +106,16 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
   /**
     * Execute a new job.
     */
-  override def execute: Future[BackendJobExecutionResponse] = Future(executeTask())
+  override def execute: Future[BackendJobExecutionResponse] = {
+    log.debug("{} Checking if hash {{}} is in the cache.", tag, jobHash)
+    cacheActor ! ReadExecutionResult(jobHash)
+    executionResponse.future
+  }
+
+  /**
+    * Abort a running job.
+    */
+  override def abort(): Unit = Future.failed(new UnsupportedOperationException("HtCondorBackend currently doesn't support aborting jobs."))
```

Comment:
I think that is not a good idea since abort may come later. It's a different call.